tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -165,3 +167,248 @@ def test_upgrade_brokers(self, from_version, to_version):
         node = self.driver.node
         node.account.ssh("grep ALL-RECORDS-DELIVERED %s" % self.driver.STDOUT_FILE, allow_fail=False)
         self.processor1.node.account.ssh_capture("grep SMOKE-TEST-CLIENT-CLOSED %s" % self.processor1.STDOUT_FILE, allow_fail=False)
+
+    @parametrize(old_version=str(LATEST_0_10_1), new_version=str(LATEST_0_10_2))
+    @parametrize(old_version=str(LATEST_0_10_1), new_version=str(LATEST_0_11_0))
+    @parametrize(old_version=str(LATEST_0_10_1), new_version=str(DEV_VERSION))
+    @parametrize(old_version=str(LATEST_0_10_2), new_version=str(LATEST_0_11_0))
+    @parametrize(old_version=str(LATEST_0_10_2), new_version=str(DEV_VERSION))
+    @parametrize(old_version=str(LATEST_0_11_0), new_version=str(DEV_VERSION))
+    def test_simple_upgrade(self, old_version, new_version):
+        """
+        Starts 3 KafkaStreams instances with <old_version>, and upgrades one-by-one to <new_version>
+        """
+
+        self.topics = {
```

Comment:
`topics` defined here and in next test, maybe move up to `init`

Reply:
The general test run with 3 broker and creates topic with replication factor 3 an other config (and used `init` for setup) -- however, we need different single-broker setup here. If we cleanup and merge tests (what we should to right away or a follow up PR -- Guozhang suggested to do it in this PR) than I agree that it makes sense to unify the setup. WDYT?